test/specs/elements/Container/Container-test.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+
+import Container from 'src/elements/Container/Container'
+import * as common from 'test/specs/commonTests'
+
+describe('Container', () => {
+  common.isConformant(Container)
+  common.rendersChildren(Container)
+  common.hasUIClassName(Container)
+
+  common.propKeyOnlyToClassName(Container, 'text')
+  common.propKeyAndValueToClassName(Container, 'aligned')
```

Comment:
I have now merged #285 which adds `useAlignedProp` propUtil and `implementsAlignedProp` common test.

The util and test account for the special handling of the `justified` value.  You should be able to update this PR now.